exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<p>Maybe it's just my habit, but I think it's great now that we have a spectrum to visualize it with a simple plt.plot() call. What would be useful about that is to also then be able to draw the reader's attention to the fact that some of the flux values in the lw spectrum are negative. I presume this is due to just large errors at the very edge of the band coupled with background subtraction. Then you can warn them, we will need to take the log of the flux which we can't do for negative numbers. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>